packit_service/service/events.py

```diff
@@ -635,3 +641,119 @@ def get_copr_build_logs_url(event: CoprBuildEvent) -> str:
         f"{event.project_name}/{event.chroot}/"
         f"{event.build_id:08d}-{event.pkg}/builder-live.log.gz"
     )
+
+
+class AbstractPagureEvent(Event, PagurePackageConfigGetter):
```

Comment:
I cannot find anything "abstract" in this class? What does the name try to suggest?

Reply:
this class is abstract - it should not be used create instances, should be used only for inheritance.

Also I followed existing architecture approach, so in case you have objection please add topic to planned arch day.